### PR TITLE
adds active elastic job gem

### LINKED ISSUE #121: Add Active Elastic Job adapter
Before move on to Elastic Beanstalk, we have to have [Active Elastic Job](https://github.com/magara/active-elastic-job) adapter to work with Elastic Beanstalk worker environment.

## Motivation

Sidekiq can be used in a worker environment; however, using SQS will be beneficial since the worker environment is designed for it.

## Pitch

Add the `active-elastic-job` gem, which will work only for the production environment.

## Alternatives

There is no alternative.

## Additional context

Please provide important parts in a pull request.

### PR DESCRIPTION
**Pull request by noloman**
- [X] Adss Active Elastic Job gem
Closes #121

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -65,6 +65,8 @@ group :production do
 
   gem 'sentry-raven'
   gem 'sidekiq', '~> 5.2.1'
+
+  gem 'active_elastic_job'
```
Comment (BerkhanBerkdemir) [inline Gemfile]: That uses the old gem, not our gem.

```suggestion
  gem 'active_elastic_job', git: 'https://github.com/magara/active-elastic-job.git'
```

**Comment by noloman** [comment]
Yes good idea because all the gems dependencies are wrong now

Op za 18 mei 2019 19:20 schreef Berkhan Berkdemir <notifications@github.com

**Comment by BerkhanBerkdemir** [comment]
Also, I will bump the Rails version of the gem, @noloman.

**Comment by noloman** [comment]
Perfect thanks. Do it on this branch and tomorrow morning I will fix the
rest

Op za 18 mei 2019 19:24 schreef Berkhan Berkdemir <notifications@github.com

**Comment by noloman** [comment]
@BerkhanBerkdemir works 😄 now.. how to test this? 🤔

**Comment by BerkhanBerkdemir** [comment]
> @BerkhanBerkdemir it works smile now.. how to test this? thinking

Well, I have no idea. This weekend I deploy this branch in an EB env.

**Comment by BerkhanBerkdemir** [comment]
Sorry @noloman, I didn't make the Beanstalk environment work. Also, for the next 2 weeks, I will be busy...

**Comment by BerkhanBerkdemir** [comment]
This PR will be resolved in version `0.2.0`, but will stay open

**Pull request closed without merging**